### ISSUE TITLE
Map doesn't expose getSize()

### ISSUE THREAD

**Issue by christophfriedrich** [MelihAltintas/vue3-openlayers#215]
The docs for the View component [say](https://vue3openlayers.netlify.app/componentsguide/view/#calculateextent-opt-size):

> calculateExtent(opt_size)
Calculate the extent for the current view state and the passed size. The size is the pixel dimensions of the box into which the calculated extent should fit. In most cases you want to get the extent of the entire map, that is map.getSize().

But the Map component doesn't expose any `getSize` method. In fact, this name appears nowhere in the entire source code except that help page.

This could also go as a feature request, but since the docs contain this, I filed it as a bug.

I stumbled across this because I want to know the bbox (in lat/long coordinates) of the area my map currently shows. Is this really _the_ way to do it? Is there any other way? I'm surprised that to stay updated on such a basic information, I apparently need to listen to the view's `onCenterChanged` and `onZoomChanged` (which fire like 50 times when I zoom a single step?!) and then calculate it myself so complicatedly. Shouldn't it be an observable property of the view?

**Comment by mathiash98** [comment]
`getSize` is a function of the OpenLayers object and not something the wrapper library `vue3-openlayers` care about. To use these functions you need to:
1. Assign the map component to a ref `<ol-map ref="map">`
2. Save the ref to a variable in the js: `const map = ref();`
3. Call `getSize()` on the map ref: `map.value.map.getSize();` -> `map.value.map` is the actual OpenLayers Map Object which this wrapper library interacts with

**Comment by d-koppenhagen** [comment]
Exactly what @mathiash98 said. Probably, I can improve the docs to make it more clear.

**Comment by christophfriedrich** [comment]
Thanks for the quick help! In fact I had tried almost exactly that (among other things), but forgot the `.value`... Now it works!

And yes, having this more precise in the docs would really help :)

Regarding my complaint that the events fire many times, I found out that this is because they also fire during animation. I switched to listening to the `map`'s `moveend` event instead, as recommended in openlayers/openlayers#7402.
